analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -158,7 +162,31 @@ private fun SpdxExternalDocumentReference.resolve(definitionFile: File, issues:
         }
     } else {
         tempDir = createOrtTempDir()
-        OkHttpClientHelper.downloadFile(uri.toString(), tempDir).getOrNull() ?: run {
+
+        val client = OkHttpClientHelper.buildClient {
+            addInterceptor { chain ->
+                val request = chain.request()
+                val requestBuilder = request.newBuilder()
+
+                // Use the authenticator also to request preemptive authentication.
+                val auth = Authenticator.requestPasswordAuthentication(
+                    /* host = */ uri.host,
+                    /* addr = */ null,
+                    /* port = */ uri.port,
+                    /* protocol = */ uri.scheme,
```

Comment:
Is it correct to use `uri.scheme` here and not two rows below?

Reply:
Yes. What `URI` calls "scheme" is the protocol, and what the `Authenticator` calls "scheme" is the "authorization scheme".